### PR TITLE
Update language-introduction.md

### PR DESCRIPTION
**Pull request by Krilivye**
I got an exception using this page: Module HelloWord does not define type HelloWord
Change proposed based on https://haxe.org/manual/introduction-hello-world.html

### CODE REVIEW

**Comment by Simn** [comment]
That doesn't seem necessary. The manual text tells you to save the file as HelloWorld.hx, in which case thing should work just fine.

**Comment by Krilivye** [comment]
My VERY very bad ... I typed class HelloWordl in HelloWorld.hx.

**Comment by Gama11** [comment]
On the other hand, `Main.hx` is a pretty common naming convention.

File: pages/documentation/introduction/language-introduction.md [hunk 1]
```diff
@@ -12,20 +12,21 @@ Hello World
 The Haxe Programming Language was designed to be simple yet powerful. Its syntax largely follows the ECMAScript standard, but deviates where necessary. The following program demonstrates "Hello World" written in Haxe:
 
 ```haxe
-class HelloWorld {
-	static public function main() {
-		trace("Hello World");
-	}
+class Main {
+  static public function main():Void {
```
Comment (ibilon) [inline pages/documentation/introduction/language-introduction.md]: I wouldn't add explicit return type here.

**Comment by andyli** [comment]
Closing since the original instruction on the manual is correct.

**Pull request closed without merging**